src/content/docs/fr/basics/layouts.mdx

```diff
@@ -237,54 +232,3 @@ const { title, fancyJsHelper } = Astro.props;
 ```
 
 <ReadMore>Pour en savoir plus sur la prise en charge de Markdown et de MDX par Astro, consultez notre [guide Markdown/MDX](/fr/guides/markdown-content/).</ReadMore>
-
-## Utilisation de TypeScript avec des mises en page
-
-Toute mise en page Astro peut être modifiée pour introduire la sécurité des types et la saisie semi-automatique en fournissant les types pour vos propriétés :
-
-```astro ins={2-7} title="src/components/MyLayout.astro"
----
-interface Props { 
-	title: string;
-  description: string;
-  publishDate: string;
-  viewCount: number;
-}
-const { title, description, publishDate, viewCount } = Astro.props;
----
-<html lang="fr">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="description" content={description}>
-    <title>{title}</title>
-  </head>
-  <body>
-    <header>
-      <p>Publié le {publishDate}</p>
-      <p>Consulté par {viewCount} personnes</p>
-    </header>
-    <main>
-      <slot />
-    </main>
-  </body>
-</html>
-```
-
-## Imbrication de mises en page
-
-Les composants de mises en pages ne doivent pas nécessairement contenir une page entière de HTML. Vous pouvez décomposer vos mises en page en composants plus petits et combiner des composants de mises en page pour créer des templates de page encore plus flexibles. Ce modèle est utile lorsque vous souhaitez partager du code entre plusieurs mises en page.
-
-Par exemple, un composant de mise en page `BlogPostLayout.astro` peut styliser le titre, la date et l'auteur d'un article. Ensuite, un `BaseLayout.astro` pour tout le site pourrait gérer le reste de votre template de page, comme la navigation, les pieds de page, les balises méta SEO, les styles globaux, et les polices de caractères. Vous pouvez également transmettre les propriétés reçues de votre message à une autre mise en page, comme tout autre composant imbriqué.
-
-```astro {3} /</?BaseLayout>/ /</?BaseLayout url={frontmatter.url}>/
----
-// src/layouts/BlogPostLayout.astro
-import BaseLayout from './BaseLayout.astro';
-const { frontmatter } = Astro.props;
----
-<BaseLayout url={frontmatter.url}>
-  <h1>{frontmatter.title}</h1>
-  <h2>Auteur du billet : {frontmatter.author}</h2>
-  <slot />
-</BaseLayout>
-```
```

Comment:
I can't add a suggestion since these lines are deleted but they shouldn't! See the [current file](https://github.com/withastro/docs/blob/73ecfb4fcebfdb443afd948f5229ed27888a309f/src/content/docs/en/basics/layouts.mdx?plain=1#L235) in `main`. (Note: there are some changes compared to the previous version)